robe.el

```diff
@@ -106,6 +107,15 @@ or another package."
            completing-read-function)))
     (apply #'completing-read args)))      ; 2) allow completing-read override
 
+(defun robe-args ()
+  (mapconcat
+   'identity
+   (loop for arg in (list
+                     (concat "host: " (if robe-host (format "'%s'" robe-host) "nil"))
```

Comment:
Why not make `robe-host` mandatory? `127.0.0.1` seems like a good default value.